File: desktop/gtk/py_basic/versus_c/simple.c
#include <gtk/gtk.h>

int main(int argc, char *argv[]){
	GtkBuilder *mybuilder;
	GtkWindow *mywindow;
	GtkButton *btnQuit;
	
	gtk_init(&argc,&argv);
	mybuilder=gtk_builder_new();
	gtk_builder_add_from_file(mybuilder,"simple.glade",NULL);
	
	mywindow=gtk_builder_get_object(mybuilder,"window1");
	gtk_window_set_default_size(mywindow,200,200);
	gtk_window_set_position(mywindow,GTK_WIN_POS_CENTER);
	gtk_window_set_title(mywindow,"simple");
	gtk_signal_connect(mywindow,"destroy",gtk_main_quit,NULL);
	
	btnQuit=gtk_builder_get_object(mybuilder,"btnQuit");
	gtk_signal_connect(btnQuit,"clicked",gtk_main_quit,NULL);
	
	gtk_widget_show(mywindow);
	gtk_main();
	
	return 0;
}

File: desktop/gtk/py_basic/versus_c/simple.py
#!/usr/bin/python

import gtk

guibuilder=gtk.Builder()
guibuilder.add_from_file("simple.glade")

window=guibuilder.get_object("window1")
window.set_default_size(200,200)
window.set_position(gtk.WIN_POS_CENTER)
window.set_title("simple")
window.connect("destroy",gtk.main_quit)

button=guibuilder.get_object("btnQuit")
button.connect("clicked",gtk.main_quit)

window.show()
gtk.main()
